chruby.el

```diff
@@ -174,6 +174,10 @@ ruby version, and the gem path"
     (if version-file-path (chruby-use (chruby--read-version-from-file version-file-path))
       (message "[chruby] could not locate .ruby-version"))))
 
+(defun chruby--available-names ()
+  "list of available ruby names without path, higher versions sorted first"
+  (reverse (cl-mapcar 'chruby-util-basename (chruby-rubies))))
```

Comment:
I don't think you need `cl-mapcar` here, regular `mapcar` should do.

From [the manual](https://www.gnu.org/software/emacs/manual/html_node/cl/Mapping-over-Sequences.html)

>  Given a single seq argument it is equivalent to mapcar; given n sequences, it calls the function with the first elements of each of the sequences as the n arguments to yield the first element of the result list, then with the second elements, and so on.